### PR TITLE
authors: submission fixes

### PR DESCRIPTION
**Pull request by jalavik**
Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

File: inspirehep/modules/authors/views.py [hunk 1]
```diff
@@ -42,20 +42,20 @@
     url_for, \
     redirect
 from flask_breadcrumbs import register_breadcrumb
+from flask.ext.login import current_user
+from flask_login import login_required
```
Comment (jacquerie) [inline inspirehep/modules/authors/views.py]: Can't this be both `from flask_login`, or am I missing something?

Comment (jalavik) [inline inspirehep/modules/authors/views.py]: Yeah, it can! Thanks!

File: inspirehep/modules/authors/views.py [hunk 2]
```diff
@@ -290,11 +291,13 @@ def submitnew():
     visitor = DataExporter()
     visitor.visit(form)
 
-    myobj = DbWorkflowObject.create_object(id_user=current_user.get_id())
-    myobj.set_data(visitor.data)
+    workflow_object = DbWorkflowObject.create_object(id_user=current_user.get_id())
+    workflow_object.data = visitor.data
+    workflow_object.save()
+    db.session.commit()
```
Comment (kaplun) [inline inspirehep/modules/authors/views.py]: Shouldn't this be part of a begin_nested?

Comment (jalavik) [inline inspirehep/modules/authors/views.py]: I don't think it is strictly required as inside the create_object and save calls there are nested transactions and as this is at top-level before the request ends we need the specific commit call.

**Comment by jalavik** [comment]
WIP awaiting new updates to invenio-workflows

**Pull request merged**